android/src/main/java/com/globo/reactnativeua/ReactNativeUA.java

```diff
@@ -51,6 +51,26 @@ public void setNamedUserId(String namedUserID) {
     }
 
     @ReactMethod
+    public void enableLocationUpdates() {
+        UAirship.shared().getLocationManager().setLocationUpdatesEnabled(true);
+    }
+
+    @ReactMethod
+    public void disableLocationUpdates() {
+        UAirship.shared().getLocationManager().setLocationUpdatesEnabled(false);
+    }
+
+    @ReactMethod
+    public void enableBackgroundLocation() {
+        UAirship.shared().getLocationManager().setBackgroundLocationAllowed(true);
```

Comment:
This will only allow location updates to continue into the background. Background location still requires `setLocationUpdatesEnabled(true)`